django/river/settings/prod.py

```diff
@@ -3,3 +3,5 @@
 # TODO: Requests should be authenticated. The following is a temporary workaround that
 #   will be removed once authencation is setup globally.
 REST_FRAMEWORK["DEFAULT_PERMISSION_CLASSES"] = ["rest_framework.permissions.AllowAny"]
+
+USE_X_FORWARDED_HOST = os.environ.get("USE_X_FORWARDED_HOST", True)
```

Comment:
can you add a comment to explain that we need to tell django that it sits behind a reverse proxy to make the redirects properly ?